asreview/settings.py

```diff
@@ -0,0 +1,30 @@
+from abc import ABC
+import os
+from asreview.utils import config_from_file, _safe_dict_update
+
+
+class ASReviewSettings(ABC):
```

Comment:
Great! Thanks. 

I don't see the reason to make this an ABC class. Inherit 'object'?